plugins/org.yakindu.sct.doc.user/src/user-guide/statechart_language.textile

```diff
@@ -46,11 +46,120 @@ States are the central elements of a state machine. A state has to be placed ins
+
+"Figure &quot;Transition syntax overview&quot;":#fig_transition-syntax-overview shows how the syntax of a transition specification is defined. You can refer to this diagram while reading the subsequent sections.
+
+p(#fig_transition-syntax-overview). 
+!(standard-image)images/docu_grammar_030_transition-overview.png(Transition syntax overview)!
+
+p=. Transition syntax overview
+
+h4(#unspecified-transition). Unspecified transition
+
+In the most simple case we have only the transition's arrow leading from a source state to a target state without any additional specification. In other words, there is no text attached to the transition. However, such an event will never be taken, so better make sure to set up a transition specification.
+
+h4(#event-triggers). Event triggers
+
+When an "event":#events occurs, it may trigger a transition to "fire". The transition's source state, which is the currently active state, becomes inactive, control is transferred along the transition's arrow to the transition's target state, and that state becomes the active state.
+
+For a transition to fire, the triggering event or events must be given in the transition specification as an _event trigger_. An event trigger consists of one or more event specifications, separated by comma. An event specification is either the name of an event that has been declared in the definition section, see "section &quot;Events&quot;":#events, or one of the built-in event specifications _after_, _every_, _always_, _default_, _else_, or _oncycle_. The latter are documented in "section &quot;Reaction triggers&quot;":#reaction-triggers.
+
+Let's have a look at some sample transition specifications. We are assuming that the transition's source state is active, because otherwise the transition wouldn't fire anyway.
+
+| *Event trigger:* | *Prerequisites for the transition to fire:* |
+| @ev@ | Event _ev_ occured. |
+| @ev_1, ev_2@ | At least one the events _ev_1_ and _ev_2_ occured. |
+| @after 3s@ | Three seconds after activating the source state have passed. |
+| @after 3s, ev@ | Three seconds have passed or event _ev_ occurred, whatever happens first. |
+| @after 3s, after 2s@ | Three seconds have passed or two seconds have passed, whatever happens first. :-) |
+| @every 3s@ | Three seconds after activating the source state have passed. |
+| @always@ | Transition fires always. |
+| @default@ | Transition fires always. |
+| @else@ | Transition fires always. |
```

Comment:
*default* and *else* are basically the same and only meaningful with choices to define a transition to be taken when all other transitions of the choice cannot be taken.